### ISSUE TITLE
Koohi stories via an API?

### ISSUE THREAD

**Issue by jregistr** [fabd/kanji-koohii#180]
Hellos!
I'm wondering, is there a way to programmatically access Kanji stories for a Kanji? Perhaps through an API?
Looking around the site I don't see any such options.
I'm mainly curious because I'd like to use them in Anki via an addon. 

Thanks!

**Comment by fabd** [comment]
Hi

I'm sorry for answering so late. Probably not helpful now but it may be helpful if the question comes again.

There is no such API, the main reason being the current codebase is too old to create a decent API. But I am not opposed to create one in the future.

I am looking at doing a "in place migration" (aka "strangler pattern"), using Laravel as the new facing framework. With a modern framework creating an API will be much more doable., in particular it needs strong caching capacity which is the main reason I'm too wary of enabling that on the current codebase (we could have a lousy API of sorts as long as it works, but the caching would be terrible).

TLDR: needs a big rewrite from symfony 1.x to current Laravel (or Symfony), and then use composrer and such to build an API the correct way.

**Comment by fabd** [comment]
Closing for now as there is no short term resolution.
